WebHostLib/tracker.py

```diff
@@ -316,6 +359,7 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int, want
     else:
         multisave: Dict[str, Any] = {}
 
+    checked_locations = multisave.get("location_checks", {}).get((tracked_team, tracked_player), set())
     # Add items to player inventory
     for (ms_team, ms_player), locations_checked in multisave.get("location_checks", {}).items():
```

Comment:
These lines seem to use the same value retrieved from the multisave. (multisave["location_checks"])

Perhaps consolidate?

Reply:
I'll have to think about this one but they aren't the same as the one I'm creating there is only pulling the checked locations for the current player while the loop is looking at checked locations of every player.